tests/Tests/ORM/Functional/PersistentCollectionTest.php

```diff
@@ -31,6 +31,15 @@ protected function setUp(): void
         PersistentObject::setObjectManager($this->_em);
     }
 
+    protected function tearDown(): void
+    {
+        if (! class_exists(PersistentObject::class)) {
```

Comment:
Is this really required? https://github.com/doctrine/orm/blob/31e6a1da87fc76dbc7273b96f9a8df6845bd7c95/tests/Tests/OrmFunctionalTestCase.php#L542-L545 does already handle this case gracefully, doesn't it?

Reply:
@xabbuh I debugged it, and `sharedConn` is always set. 

Moreover, using this code:

```
    protected function tearDown(): void
    {
        try {
            parent::tearDown();
        } catch (\Throwable $e) {
            var_dump($e->getMessage());
        }
    }
```

With PHPUnit _10_
prints this:

```
string(41) "Call to a member function clear() on null"
```

So the same issue exists with both versions of PHPUnit, it's just that for some reason, PHPUnit 10 swallows the exception.

Reply:
I replaced the commit that introduce this change with another that acts at the `OrmFunctionalTestCase` level, by also checking `$this->_em`.

Reply:
Yes, nice find!